try: 
	print("hello")
	print("world")
except: 
	print('an error occured in try_trial.py')